src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: gray;
+  text-transform: uppercase;
```

Comment:
we don't want all texts to become uppercase, plese add this style to nav__link to make only whose elements uppercase